### PR TITLE
fix(adhoc): prevent meeting join for non-whitelisted domains 

### PR DESCRIPTION
**Pull request by virtuacoplenny**
Screen Shot 2019-12-08 at 1 22 28 PM

### CODE REVIEW

**Comment by virtuacoplenny** [comment]
Looks like I'll have to figure out how to fix the selenium tests to check the cancel-join button. Probably while I'm at it I can write a new selenium test for the changes in this pr.

File: spot-client/src/common/utils/meeting.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { findWhitelistedMeetingUrl } from './meeting';
+describe('meeting utils', () => {
+    describe('findWhitelistedMeetingUrl', () => {
```
Comment (paweldomas) [inline spot-client/src/common/utils/meeting.test.js]: any tests that would cover the exclude pattern?

Comment (virtuacoplenny) [reply spot-client/src/common/utils/meeting.test.js]: Do you know why the exclude pattern even exists? I can't remember.

Comment (virtuacoplenny) [reply spot-client/src/common/utils/meeting.test.js]: The jitsi dial in info page...that's why.

**Pull request merged**